runtime/caml/memory.h

```diff
@@ -155,6 +155,14 @@ CAMLextern wchar_t* caml_stat_wcsdup(const wchar_t *s);
 CAMLextern wchar_t* caml_stat_wcsdup_noexc(const wchar_t *s);
 #endif
 
+/* [caml_stat_strndup(s, n, &out_n)] returns a copy of the first n bytes of s.
+   If out_n is not NULL, then the size of the result is recorded in *out_n. This
```

Comment:
```suggestion
/* [caml_stat_strndup(s, len, &out_len)] returns a copy of the first len bytes of s.
   If out_len is not NULL, then the size of the result is recorded in *out_len. This
```
(for consistency with the argument names in the prototype below)